### DISCUSSION TITLE
Li

### DISCUSSION THREAD

**Discussion by albertosantamariapang** [POSTECH-CVLab/PyTorch-StudioGAN#35] [General] [answered]
The link (https://tiny-imagenet.herokuapp.com/) to the TinyImageNet dataset is broken. I wonder if can you provide an updated link.

**Comment by mingukkang** [comment, marked as answer]
Hi,
 
You can download Tiny ImageNet dataset by typing "wget http://cs231n.stanford.edu/tiny-imagenet-200.zip" in the terminal.
Thank you.

Sincerely, 

Minguk Kang
